corgi/tasks/errata_tool.py

```diff
@@ -125,16 +152,34 @@ def slow_load_errata(erratum_name: str, force_process: bool = False) -> None:
             )
 
     if force_process:
-        slow_save_errata_product_taxonomy.delay(erratum_id)
+        for build_id in build_variants.keys():
+            sb = SoftwareBuild.objects.get(build_id=build_id, build_type=build_type)
+            sb.save_product_taxonomy()
 
     logger.info("Finished processing %s", erratum_id)
 
 
-def _get_relation_builds(erratum_id: int) -> QuerySet:
+def _get_relation_builds(erratum_id: int) -> Tuple[dict[str, list[str]], str]:
     # Get all the PCR with errata_id
-    return ProductComponentRelation.objects.filter(
+    relation_builds = ProductComponentRelation.objects.filter(
         type=ProductComponentRelation.Type.ERRATA, external_system_id=erratum_id
-    ).values_list("build_id", "build_type")
+    ).values_list("build_id", "build_type", "product_ref")
+    build_variants: dict[str, list[str]] = defaultdict(list)
+    build_types = set()
+    # Check erratum builds all have the same type
+    for build_id, build_type, variant_id in relation_builds:
+        build_variants[build_id].append(variant_id)
+        build_types.add(build_type)
+
+    # Errata are not used in community products, and we are not yet attaching other builds types
+    # like HACBS to errata. If that changes, we should review slow_load_errata for correctness
+    if len(build_types) > 1:
+        raise ValueError("Multiple build types found for errata %s", erratum_id)
+    elif len(build_types) == 0:
+        build_type = BUILD_TYPE
```

Comment:
It took me a minute to figure out the 0 build_types case exists, so let's add a comment here since it's not immediately obvious. 
```suggestion
        # if the queryset above had no results
        build_type = BUILD_TYPE
```